data/prophets_stake.json

```diff
@@ -0,0 +1,1259 @@
+[
```

Comment:
we don't need objects. Just list of addresses ['0x...','0x...'] and we don't need a PR a gist or a file is fine